minio-client/main.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	minio "github.com/minio/minio-go"
+)
+
+var (
+	accessKeyID              = os.Getenv("ACCESS_KEY")
+	secretAccessKey          = os.Getenv("SECRET_KEY")
+	serviceName              = os.Getenv("SERVICE_NAME")
+	namespace                = os.Getenv("NAMESPACE")
+	port                     = os.Getenv("PORT")
+	livenessCheckInterval, _ = strconv.Atoi(os.Getenv("LIVENESS_CHECK_INTERVAL"))
+	livenessRetryCount, _    = strconv.Atoi(os.Getenv("LIVENESS_RETRY_COUNT"))
+	mode                     = os.Getenv("MODE")
+	capacity, _              = strconv.ParseFloat(os.Getenv("CAPACITY"), 32)
+)
```

Comment:
Can we have error handling for unavailability of these, i.e., log saying what is missing --- as all these are mandatory ENVs

Reply:
PTAL to new changes.

Reply:
Just thinking..would it make more sense to have:

- a list with mandatory env keys pre-defined and check if they are a subset of current session's [list](https://gist.github.com/nickmccurdy/9519543) & that they are initialized to non-empty values.